consensus/src/fallback.rs

```diff
@@ -143,7 +158,7 @@ impl Fallback {
         } else if qc.fallback == 1 {
```

Comment:
Is there any other value than {0,1}? maybe remove else if so that it breaks something if it happens.